package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@proximaone/stream-client-js",
+  "name": "@proximaone/streamdb-client-js",
```

Comment:
such a major change for a documentation PR. Please, leave it as `stream-client-js` which already has a lot of backlinks in other projects